pkg/registry/core/pod/strategy.go

```diff
@@ -686,7 +687,7 @@ func applyWaitingForSchedulingGatesCondition(pod *api.Pod) {
 	pod.Status.Conditions = append(pod.Status.Conditions, api.PodCondition{
 		Type:    api.PodScheduled,
 		Status:  api.ConditionFalse,
-		Reason:  api.PodReasonSchedulingGated,
+		Reason:  apiv1.PodReasonSchedulingGated,
```

Comment:
Any side effect of changing it from internal or external types? (given this file belongs to apiserver cc @ahg-g )